cmd/zed/index/create.go

```diff
@@ -20,9 +20,22 @@ import (
 
 var create = &charm.Spec{
 	Name:  "create",
-	Usage: "create [options] rule-name pattern",
+	Usage: "create [options] rule-name rule-type value",
 	Short: "create an index rule for a lake",
-	New:   newCreate,
+	Long: `The index create command creates and index rule that can be applied to any
```

Comment:
Other `charm.Spec.Long` values begin with a newline. And s/and/an/.
```suggestion
	Long: `
The index create command creates an index rule that can be applied to any
```